addOns/webuipoc/webuipoc.gradle.kts

```diff
@@ -41,6 +41,19 @@ for (dir in pocsSrcDir.listFiles()!!) {
                 }
             },
         )
+
+if (normalizedPocName == "ReactWebUI") {
++            val lintTask =
```

Comment:
The `+` should be removed, that's part of the diff syntax.

Reply:
done

Reply:
You might need to run the Gradle `spotlessApply` task.

Reply:
applied